.circleci/config.yml

```diff
@@ -6,9 +6,10 @@
 version: 2.1
 
 orbs:
-  # This is needed to be able to install Node.js in the primary container,
-  # which provides python only.
-  node: circleci/node@4.7.0
+  # This is needed to be able to install Node.js 14 in the primary container,
+  # which provides python only (the convenience cimg/python:3.9-node provides
+  # node 16).
+  node: circleci/node@5.0.0
```

Comment:
This is just an orb, so it doesn't need to change. I updated the version but it shouldn't be any different, we're explicitly installing a specific nodejs version.

Reply:
_could_ it change though?  Using the deprecated images seems undesirable, even if it's not an immediate problem.

Reply:
It's (confusingly) not an image, just a (very fancy) bit of config. It isn't deprecated like the `circleci/` images are, and doesn't have a replacement.

Reply:
Oh! got it